### PR TITLE
#160: Added system test that ensures the different paths are recommen…

### PR DESCRIPTION
**Pull request by thomasgauvin**
…ded based on accessibility settings

I have the wrong issue number, but this is related to making the path algorithm take into account the accessibility/staff access.

This is done through our screenshot testing facilitating methods, and it's really cool. Woop woop!

### CODE REVIEW

**Comment by thomasgauvin** [comment]
image (1)

**Comment by thomasgauvin** [comment]
All system tests passing:

image

File: app/src/androidTest/java/com/example/concordia_campus_guide/EspressoHelpers.java [hunk 1]
```diff
@@ -79,4 +90,79 @@ public static void cancelNotificationIfExists(){
...
+
+        Bitmap bmp = bm1.copy(bm1.getConfig(), true);
+        int threshold = 10;
+        boolean isSame = true;
+
+        //use coordinates to reduce flakiness of screenshot testing
+        int x1 = 470;
+        int x2 = 618;
+        int y1 = 989;
+        int y2 = 1139;
```
Comment (Hambrsoom) [inline app/src/androidTest/java/com/example/concordia_campus_guide/EspressoHelpers.java:136]: Do you think we can use more representative names?

Comment (thomasgauvin) [reply app/src/androidTest/java/com/example/concordia_campus_guide/EspressoHelpers.java:136]: these are x,y coordinates on the screenshot to make the triangle, do you have any ideas for more representative names?

**Pull request merged**